migration-engine/core/Cargo.toml

```diff
@@ -25,6 +25,6 @@ url = "2.1.1"
 tokio = { version = "1.0", default-features = false, features = ["macros"] }
 
 [features]
-default = ["sql"]
+default = ["sql", "mongodb"]
```

Comment:
nitpick: do we need it the ME for releases? My outsider understanding is that people are only interested about testing mongo with the client, for now

Reply:
Maybe, maybe not. It has minimal tooling to create and drop things. If it's a problem for the ME team that the dependency is added due to size, we can probably drop it.

Reply:
I guess we can just add it, and remove it if it causes any problem